packages/create-sitecore-jss/src/templates/nextjs/graphql-codegen.yml

```diff
@@ -0,0 +1,9 @@
+overwrite: true
+# documents: 'src/**/*.graphql'
```

Comment:
Can you add a comment explaining why is this commented out, and when devs might need to enable it?

Reply:
@yavorsk If it will be disabled by default, it means that in case we scaffold _nextjs-styleguide_ - graphql code generation will not be executed, so the error will be thrown in GraphQL-ConnectedDemo, did you test that?